src/RetireNet.Packages.Tool/RetireNet.Packages.Tool.csproj

```diff
@@ -15,9 +14,12 @@
         <license>https://raw.githubusercontent.com/RetireNet/dotnet-retire/master/LICENSE</license>
         <RepositoryUrl>https://github.com/RetireNet/dotnet-retire</RepositoryUrl>
         <PackageIconUrl>https://raw.githubusercontent.com/RetireNet/images/master/nuget/dotnet-retire/icon.512x512.png</PackageIconUrl>
+        <Icon>images\icon.512x512.png</Icon>
         <RepositoryType>git</RepositoryType>
+        <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;netcoreapp2.2</TargetFrameworks>
     </PropertyGroup>
     <ItemGroup>
+        <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.1.0" />
```

Comment:
Why is this needed?

Reply:
It's my easy button for disabling the screen output from the tool that I don't need in the build. If there's another way to change the LogLevel at runtime, I couldn't sus out what it was faster than I knew how to make it read from the environment variables :)

Reply:
Ah, ok. [Pr readme](https://github.com/RetireNet/dotnet-retire/blob/a060df1dcd76a390c4ac179cee9af7e26aa59a00/README.md#usage), you can just pass an argument `--loglevel={Trace|Debug|Information|Warning|Error|Critical}`.

Reply:
Huh. Missed that. And I missed seeing this in the code:

```
            var logLevel = config.GetValue<LogLevel>("LogLevel");
```

Doh! hah.